modals/modals.css

```diff
@@ -0,0 +1,27 @@
+.modal {
+    box-shadow: #00000059 0px 5px 15px;
+    border: 1px solid var(-lynxWhite);
+    background-color: var(--lynxWhite);
+    border-radius: var(--border-sm);
+    width: fit-content;
+    padding: 1rem 2rem;
+    display: flex;
+    flex-direction: column;
```

Comment:
can make separate class for this

Reply:
Yess I have note down all point. do it in final refactor